SwiftVocabulary/SwiftVocabulary/DefinitionVC.swift

```diff
@@ -0,0 +1,30 @@
+//
+//  DefinitionVC.swift
+//  SwiftVocabulary
+//
+//  Created by Norlan Tibanear on 5/20/20.
+//  Copyright © 2020 Norlan Tibanear. All rights reserved.
+//
+
+import UIKit
+
+class DefinitionVC: UIViewController {
+    
+    var vocabWords: VocabularyWord?
+    
+    @IBOutlet var titleLabel: UILabel!
+    @IBOutlet var definitionLabel: UITextView!
```

Comment:
Minor nitpick, I would name this definitionTextView so that it is clear that it is a UITextView and not a UILabel